src/WWT.Web/Startup.cs

```diff
@@ -101,6 +102,9 @@ public void ConfigureServices(IServiceCollection services)
                 builder.AddFilter("Swick.Cache", LogLevel.Trace);
                 builder.AddDebug();
             });
+
+            services.AddSnapshotCollector((snapshotConfiguration) => 
+                _configuration.Bind(nameof(SnapshotCollectorConfiguration), snapshotConfiguration));
```

Comment:
I am against the `nameof(...)` use with config block names as the name of the variable isn't really tied to the config section name.

Reply:
I would encourage you to share that feedback with our official docs on this configuration: https://docs.microsoft.com/en-us/azure/azure-monitor/app/snapshot-debugger-vm, the nameof approach is what we use to inform customers on how to set this up.

Reply:
Ah that's a shame. I'll provide some feedback there.